examples/xgboost/tree-based/job_configs/cyclic_base/app/config/config_fed_client.json

```diff
@@ -12,7 +12,7 @@
         "args": {
           "data_split_filename": "data_split.json",
           "training_mode": "cyclic",
-          "num_tree_bagging": 5,
+          "num_client_bagging": 1,
```

Comment:
this change value from 5 to 1 would it affect the algorithm? @ZiyueXu77

Reply:
it should be default to 1 since it is cyclic (previous example makes the change during config_gen)